### PR TITLE
451006 - YauheniiKhudzik

### PR DESCRIPTION
**Pull request by MartinEmbers**
Please, take a look

### CODE REVIEW

File: DtoGenerator/CodeGenerators/ICodeGenerator.cs [hunk 1]
```diff
@@ -0,0 +1,7 @@
+﻿namespace DtoGenerator.CodeGenerators
+{
+    public interface ICodeGenerator
+    {
+        void GenerateCode(object generatingClass);
```
Comment (RamanBut-Husaim) [inline DtoGenerator/CodeGenerators/ICodeGenerator.cs]: @MartinEmbers Why this class accepts `object` parameter? As far as I see you always cats it to the appropriate representations across your code. Is it possible to accept a specific value or make it generic?

**Comment by RamanBut-Husaim** [comment]
@MartinEmbers Approved.

**Comment by MartinEmbers** [comment]
All requested changes have been completed. Please, take a look

**Comment by MartinEmbers** [comment]
All requested changes have been completed. Please, take a look

**Comment by MartinEmbers** [comment]
Ready to merge

**Comment by RamanBut-Husaim** [comment]
@MartinEmbers Accepted.

**Pull request merged**